### PR TITLE
More flexibility when defining EEI dependencies for EVM opcodes

### PR DESCRIPTION
**Pull request by jwasinger**
`CALLDATALOAD` uses an import `callDataCopy256` which is an EEI method that has been deprecated.

This PR contains a refactor that will allow EVM opcodes to directly specify the EEI dependencies they need.  It also fixes `CALLDATALOAD`.

### CODE REVIEW

**Comment by axic** [comment]
> However, `CALLDATALOAD` uses an import `callDataCopy256` which is an EEI method that has been deprecated.

Oh well actually let me remember. I think the reasoning was that `CALLDATALOAD` has a way different price to `CALLDATACOPY` which could be replicated properly via the generic `callDataCopy` :)

File: wasm/generateInterface.js [hunk 1]
```diff
@@ -167,6 +203,10 @@ function generateManifest (interfaceManifest, opts) {
   for (let opcode in interfaceManifest) {
     const op = interfaceManifest[opcode]
       // generate the import params
+    if (opcode == 'CALLCODE') {
+      debugger
```
Comment (axic) [inline wasm/generateInterface.js]: Why is this needed?

Comment (jwasinger) [reply wasm/generateInterface.js]: Leftover artifact from testing on my own machine

**Comment by axic** [comment]
Rebased.

**Comment by axic** [comment]
@jwasinger is this PR still relevant?

**Comment by jwasinger** [comment]
@axic I don't think so.
